Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -2494,7 +2494,6 @@ header: <WebCore/ScreenProperties.h>
     bool screenHasInvertedColors;
     bool screenSupportsHighDynamicRange;
 #if HAVE(SUPPORT_HDR_DISPLAY)
-    bool suppressEDR;
     float currentEDRHeadroom;
     float maxEDRHeadroom;
```

Comment:
Why do these things that depend on application foreground status need to be in a `ScreenProperties` isn the first place? That's what is causing these problems. Can't the suppression status/current headroom be passed separately as a separate message that would only do the required work and not all the work associated with screen property changes?

Reply:
We could move currentEDRHeadroom out of ScreenProperties but would likely want to discuss with @mattwoodrow first